parsers/barcode_field_defaults.go

```diff
@@ -0,0 +1,33 @@
+package parsers
+
+import (
+	"strconv"
+
+	"github.com/ingridhq/zebrash/printers"
+)
+
+func NewBarcodeFieldDefaults() *CommandParser {
+	const code = "^BY"
+
+	return &CommandParser{
+		CommandCode: code,
+		Parse: func(command string, printer *printers.VirtualPrinter) (interface{}, error) {
+			parts := splitCommand(command, code, 0)
+			if len(parts) > 0 {
+				if v, err := strconv.Atoi(parts[0]); err == nil {
+					printer.DefaultBarcodeModuleWidth = v
+				}
+			}
+			// TODO: We should be parsing WideBarToNarrowBarWidthRatio from parts[1], but:
+			// 1. The libs we use for encoding barcodes does not support this
+			// 2. We don't need it at the moment
```

Comment:
let's maybe still parse it, just don't implement it in renderer

Reply:
That's what I did initially, but I ended up with an additional, unused field `WideBarToNarrowBarWidthRatio` defined on the VirtualPrinter. I think this approach does not clutter our code and it's intent is more clear.

Reply:
If you have them group them in one struct it will be barely noticeable if one of those fields is unused

Reply:
But then what is even the point of having this field if it's unused? We would be ignoring the parsing error anyway, so it does not affect the logic at all.

Reply:
Beats me, it is just TODO that bothers me
If we can complete the parser it is better to complete it while leaving some parts of renderer unimplemented

Reply:
fine